stock_exchange/apps/stock/models.py

```diff
@@ -1,4 +1,77 @@
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from datetime import date
 
+from stock_exchange.apps.users.models import User
 
-# Add your models here
\ No newline at end of file
+#This tuple is used for choice option for
+#inform users about market operation status.
+MARKET_OPTIONS = {
+    ('B', 'buy'),
+    ('S', 'sell')
+}
+
+class Currency(models.Model):
+    """
+    Storage for information about used
+    currency in stock.
+    """
+    
+    name = models.CharField(
```

Comment:
This looks bad IMO it's better like: `name = models.CharField(_('Nazwa waluty'), max_length = 100)`

Reply:
Add some docstrings about those fields please